core/modules/cart/store/actions.ts

```diff
@@ -91,9 +92,9 @@ const actions: ActionTree<CartState, RootState> = {
     }
   },
   serverTotals (context, { forceClientState = false }) { // pull current cart FROM the server
-    if (rootStore.state.config.cart.synchronize_totals && !Vue.prototype.$isServer) {
+    if (rootStore.state.config.cart.synchronize_totals && !isServer) {
```

Comment:
in future we'd like to get rid of config inside state because it is ~16kB of data  which is barely needed on initial load. Please import `config` and apply suggestion
```suggestion
    if (config.cart.synchronize_totals && !isServer) {
```